.github/workflows/main.yml

```diff
@@ -4,20 +4,53 @@ on:
   push:
     branches:
       - main
-    paths-ignore:
-      - '**.md'
   pull_request:
     types:
       - opened
       - synchronize
-    paths-ignore:
-      - '**.md'
 
 env:
   BUNDLE_PATH: vendor/bundle
 
 jobs:
-  lint:
+  lint-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: dorny/paths-filter@v2
```

Comment:
I saw somewhere a syntax for semi-native GH Actions conditions, but lost it and didn't find.